packages/fxa-auth-server/CHANGELOG.md

```diff
@@ -1,34 +1,38 @@
+## 1.145.1
+
+No changes.
+
 ## 1.145.0
 
 ### New features
 
-* support form: add subscription plan dropdown (bf89c002d)
-* codes: Add the ux for signup codes (37929e13a)
-* fortress: add "fortress" relying party (2ffe1334d)
+- support form: add subscription plan dropdown (bf89c002d)
```

Comment:
All these changes were automatic and are consistent with the rest of the changelog.